sphinx/themes/basic/static/doctools.js

```diff
@@ -252,6 +253,31 @@ var Documentation = {
     });
     var url = parts.join('/');
     return path.substring(url.lastIndexOf('/') + 1, path.length - 1);
+  },
+
+  /**
+   * init onKeyListernes (for navigation)
+   */
+  initOnKeyListeners: function() {
+    $(document).keyup(function(event) {
+      var activeElementType = $(document.activeElement).prop('tagName');
+      if (activeElementType !== 'TEXTAREA' && activeElementType !== 'INPUT') { // don't navigate when in search box or textarea
```

Comment:
Another thing that occurred to me:  What about `<select>`?

Also:  Please move the comment to another line to satisfy 80 columns requirement.